### PR TITLE
Fix the rendering issue, refine the content.

### PR DESCRIPTION
**Pull request by liweintu**
This PR is to fix the rendering issue and refine the content.

### CODE REVIEW

**Comment by liweintu** [comment]
The rendering issue is fixed now. It's still a WIP as the content needs refinement.

@Tankya2 , @Vinitha-balachandran Could you help add a bit more detailed content to make the doc a reasonably basic guide. Also, some code block does not match with the description above, please refine the code block as well.

**Comment by scarrazza** [comment]
@liweintu @Vinitha-balachandran as you know, we have to release this package as soon as possible, therefore please apply your changes asap and make this PR ready for review.

**Comment by liweintu** [comment]
> @liweintu @Vinitha-balachandran as you know, we have to release this package as soon as possible, therefore please apply your changes asap and make this PR ready for review.

Yes, we're working on it now. Thanks for the reminder and help of the commits.

File: src/qibotn/circuit_convertor.py [hunk 1]
```diff
@@ -105,13 +131,36 @@ def init_intermediate_circuit(self, circuit):
...
+
+        This method initializes a basis map for the quantum circuit, which maps binary
+        strings representing qubit states to their corresponding quantum state vectors.
+
+        Parameters:
+            backend (object): The backend object providing the array conversion method.
+            dtype (object): The data type for the quantum state vectors.
+
+        Returns:
+            None
```
Comment (alecandido) [inline src/qibotn/circuit_convertor.py:144]:
```suggestion
```

File: src/qibotn/circuit_convertor.py [hunk 2]
```diff
@@ -79,11 +88,28 @@ def _parse_gates_to_mode_labels_operands(
...
+        """Initialize the intermediate circuit representation.
+
+        This method initializes the intermediate circuit representation by extracting gate matrices and qubit IDs
+        from the given quantum circuit.
+
+        Parameters:
+            circuit (object): The quantum circuit object.
+
+        Returns:
+            None
```
Comment (alecandido) [inline src/qibotn/circuit_convertor.py:111]:
```suggestion
```

File: src/qibotn/circuit_convertor.py [hunk 3]
```diff
@@ -26,6 +26,15 @@ def __init__(self, circuit, dtype="complex128"):
         self.circuit = circuit
 
     def state_vector_operands(self):
+        """Create the operands for dense vector computation in the interleave
+        format.
+
+        Parameters:
+            None
+
```
Comment (alecandido) [inline src/qibotn/circuit_convertor.py:34]:
```suggestion
```

**Review by alecandido — APPROVED** [review]

File: src/qibotn/circuit_to_mps.py [hunk 4]
```diff
@@ -7,6 +7,18 @@
...
+    """A helper class to convert Qibo circuit to MPS.
+
+    Parameters:
+        circ_qibo: The quantum circuit object.
+        gate_algo(dict): Dictionary for SVD and QR settings.
+        datatype (str): Either single ("complex64") or double (complex128) precision.
+        rand_seed(int): Seed for random number generator.
+
+    Return:
+        None.
```
[the remark this answers is not in the document]
Comment (Tankya2) [reply src/qibotn/circuit_to_mps.py]: Have remove all None. Thanks!

**Review by scarrazza — APPROVED** [review]

**Pull request merged**